app/poros/activity.rb

```diff
@@ -0,0 +1,29 @@
+class Activity
+  attr_reader :id, :name, :address, :lat, :lon, :rating, :time, :expenses, :description
+
+  def initialize(id, name, address, lat, lon, rating, time, expenses, description)
+    @id = id
+    @name = name
+    @address = address
+    @lat = lat
+    @lon = lon
+    @rating = rating
+    @time = time
+    @expenses = expenses
+    @description = description
+  end
+
+  def self.from_json(json)
+    new(
```

Comment:
I think this is supposed to be done in the Facade?

Reply:
I actually never use Facade in any of my projects because I think it is just redundant and make things more complicated since we dont have many filtering. It is my preference. And on my other projects, when I talked to Juliet, she was fine when I used the services for everything.

Reply:
😂  nice!!